docs/tutorials/cpp.md

```diff
@@ -72,14 +104,23 @@ int main() {
 }
 ```
 
+You can build this with `g++ main.cpp icu/target/release/libicu.a`. C++ versions beyond C++17 are supported, as are other C++ compilers.
```

Comment:
Nit: Although directly passing the `.a` file path works, I find it better style to instead pass them as libraries, like `g++ -static main.cpp -lmyicu4x -Lmyicu4x/target/release`

Reply:
why use three arguments and let the compiler find the library if one argument does the trick?